CHANGELOG.adoc

```diff
@@ -9,8 +9,12 @@ The format is based on https://keepachangelog.com/en/1.0.0/[Keep a Changelog].
 
 = 2021
 
-== 0.1.X - 2021-10-08
+== 0.1.X - 2021-11-08
+=== Security
+
+* Bumped guava and jsoup to address CVE vulnerabilities: CVE-2020-8908, CVE-2021-37714
 
+== 0.1.X - 2021-10-08
```

Comment:
i'm a bit confused by these headers, it looks like there is two now for `0.1.X`? could the new `Security` section go under the existing header for `0.1.X` instead?

Reply:
I'd combine them. If we want to keep the date, the date can go to individual changelog.